lib/plugins/features/movies/controller.js

```diff
@@ -8,3 +8,27 @@ exports.create = (payload) => {
     return new Movie({ id: movie.id }).fetch();
   });
 };
+
+exports.list = (queryOptions) => {
```

Comment:
optional: use `filter` or `query` instead of `queryOptions`. it's what we name the argument for list endpoints in api.